src/Symfony/Component/Messenger/Retry/MultiplierRetryStrategy.php

```diff
@@ -76,7 +76,7 @@ public function isRetryable(Envelope $message, \Throwable $throwable = null): bo
     /**
      * @param \Throwable|null $throwable The cause of the failed handling
      */
-    public function getWaitingTime(Envelope $message, \Throwable $throwable = null): int
+    public function getWaitingTime(Envelope $message, \Throwable|null $throwable = null): int
     {
```

Comment:
This can be reverted as the default value already holds the info (same in other places)